content/configuration/index.md

```diff
@@ -1,121 +1,120 @@
-# Change resources related settings
-kubectl edit configmap botkube-global-config -n botkube
-```
-
-```bash
-# Change communication related settings
-kubectl edit secret botkube-communication-secret -n botkube
-```
-
-This command opens ConfigMap `specs` in default editor. Do the required changes, save and exit. The BotKube Pod will automatically restart to have these configurations in effect.
-
-## Helm install options
-
-Advanced Helm install options are documented [here](/configuration/helm-chart-parameters).
-
-## Environment variables
-
-The individual communication settings can be specified via environment variables. They take priority and override the configuration specified in the file.
-
-To construct the environment variable name, take any property from the configuration file and make it uppercase. Use the underscore for properties that are nested. Use the double underscore for all camelCase properties. Finally, add the `BOTKUBE_` prefix.
-
-For example, such configuration property from YAML:
-
-```yaml
-settings:
-  kubectl:
-    defaultNamespace: 'NAMESPACE'
-```
-
-is mapped to the `BOTKUBE_SETTINGS_KUBECTL_DEFAULT__NAMESPACE` environment variable.
-
-This is a useful feature that allows you to store the overall configuration in a file, where sensitive data, such as tokens, can be put in environment variables. See the [**Tokens from Vault via CSI driver**](/configuration/communication/vault-csi/) tutorial for an example use-case.
-
-## Merging strategy
-
-BotKube allows you to split individual settings into multiple configuration files. The following rules apply:
-- The priority will be given to the last (right-most) file specified.
-- Objects are merged together and primitive fields are overridden. For example:
-  ```yaml
-  # a.yaml - first file
-  settings:
-    clustername: dev-cluster
-    configWatcher: true
-    kubectl:
-      enabled: false
-  ```
-  ```yaml
-  # b.yaml - second file
-  settings:
-    kubectl:
-      enabled: true
-  ```
-
-  ```yaml
-  # result
-  settings:
-    clustername: dev-cluster
-    configWatcher: true
-    kubectl:
-      enabled: true
-  ```
-
-- The arrays items are not merged, they are overridden. For example:
-
-  ```yaml
-  # a.yaml
-  settings:
-    kubectl:
-      enabled: true
-      commands:
-        verbs: ["api-resources", "api-versions", "cluster-info", "describe", "diff", "explain", "get", "logs", "top", "auth"]
-  ```
-  ```yaml
-  # b.yaml
-  settings:
-    kubectl:
-      commands:
-        verbs: ["get", "logs", "top", "auth"]
-  ```
-
-  ```yaml
-  # b.yaml
-  settings:
-    kubectl:
-      enabled: true
-      commands:
-        verbs: ["get", "logs", "top", "auth"]
-  ```
+---
+id: configuration
+title: Configuration
+sidebar_position: 2
+---
+
+BotKube backend allows you to specify [resource](/configuration/resource), [communication](/configuration/communication), and [global](/configuration/global) settings. Check the related documents for more detailed explanation.
```

Comment:
```suggestion
BotKube backend allows you to specify [resource](resource.md), [communication](./communication), and [global](./global.md) settings. Check the related documents for more detailed explanation.
```

Please update all links in this way.

Reply:
The same here fixe here https://github.com/madebyrogal/botkube-docs/commit/061cbbf1ee75fb4d2cdd3c7ab79578e81d32997a